src/app/libmain.cxx

```diff
@@ -103,10 +103,8 @@ static RpmOstreeCommand commands[] = {
   /* Rust-implemented commands; they're here so that they show up in `rpm-ostree
    * --help` alongside the other commands, but the command itself is fully
```

Comment:
Oh wait, I somehow missed that we had this!  Is it that easy?

Reply:
Yup, works fine! The user can't actually call the C entrypoint since it gets handled earlier so we can just leave the function as `NULL`.